### DISCUSSION TITLE
Prisma Client - model dont appear in client after generate // The underlying table does not contain a valid unique identifier and can therefore currently not be handled by Prisma Client

### DISCUSSION THREAD

**Discussion by lo50nrth** [prisma/prisma#24445] [Q&A] [answered]
### Question

Hello,

i pulled the db schema.prisma out of our SQL DB:
```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "sqlserver"
  url      = env("DATABASE_URL")
}

[...]
```
![image](https://github.com/prisma/prisma/assets/169528393/0dde823f-3e34-494f-881f-5b31e0eab3cf)

![image](https://github.com/prisma/prisma/assets/169528393/f53cc00c-482a-43e2-9be4-8ea70a668e4a)

After executing the `npx prisma generate` command, I was able to access the ‘ITHArtikelVariantenVersandeinheiten’ with the API.

To make the other model ITHArtikelVariantenPackstuecke also accessible for the client, I adapted the model as follows:

![new model](https://github.com/prisma/prisma/assets/169528393/2d15609c-4198-4b55-bbc4-47bf53e675fe)

but i had no success and no access to the model with prisma client.

### Expected behavior (optional)

I thought after that i can generate the new prisma schema in my prisma client. but i dont have access with my API for this model:
![image](https://github.com/prisma/prisma/assets/169528393/3fb59539-8f97-4319-a441-010a4b7a77c3)
error: Property 'ITHArtikelVariantenPackstuecke' does not exist on type 'PrismaClient<PrismaClientOptions, never, DefaultArgs>'.ts(2339)

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @lo50nrth 👋 

Thank you for raising this question.

Please reload your Typescript Server in VSCode or try restarting the VSCode IDE. There have been instances where a simple restart fixed this kind of issue. Do let us know if that fixes it for you.

**Reply by lo50nrth** [reply]
Hi @ludralph 
thank you for this suggested solution.

I have tried both solutions, but I still get the same error message.

**Reply by janpio** [reply, marked as answer]
You have to lowercase the first letter of the model name. Your autocomplation on `prisma.|` should suggest that.

**Reply by lo50nrth** [reply]
Thank you very much. I have written the letter in lower case and restarted the Typescript server --> error fixed
